consensus/validator.go

```diff
@@ -199,16 +200,21 @@ func (consensus *Consensus) onPrepared(msg *msg_pb.Message) {
 	if bytes.Equal(consensus.blockHash[:], emptyHash[:]) {
 		copy(consensus.blockHash[:], blockHash[:])
 	}
-	blockNumBytes := make([]byte, 8)
-	binary.LittleEndian.PutUint64(blockNumBytes, consensus.blockNum)
+	commitPayload := signature.ConstructCommitPayload(consensus.ChainReader,
+		consensus.blockHash, consensus.epoch, consensus.blockNum, consensus.viewID)
```

Comment:
use the viewID from the recvMsg. Local viewID may be not consistent with others.

Reply:
changed in latest to use received viewID